src/sage/matroids/flats_matroid.pxd

```diff
@@ -7,7 +7,8 @@ cdef class FlatsMatroid(Matroid):
     cpdef groundset(self)
     cpdef _rank(self, X)
     cpdef full_rank(self)
-    cpdef _is_independent(self, F)
+    cpdef _closure(self, X)
+    cpdef _is_closed(self, X)
```

Comment:
```suggestion
    cpdef bint _is_closed(self, X)
```
Perhaps it would be good to start adding such return type indicators in some key places? We probably have to see if the C code simplifies in some places.

Reply:
I am planning this on a massive scale on a future PR.